CONTRIBUTING.md

```diff
@@ -0,0 +1,243 @@
+# 🎭 Contributing to Theater Kid
+
+Thank you for your interest in contributing to Theater Kid! This guide will help you get started with contributing to our modern Angular chat application for AI conversations.
+
+## 🚀 Getting Started
+
+### Prerequisites
+
+- **Node.js LTS (v20+)**: Latest stable Node.js runtime
```

Comment:
If you're going to specify `LTS` then let's not specify a specific number. Get rid of "(v20+)"